tests/test_URDFParser.py

```diff
@@ -0,0 +1,137 @@
+import unittest
+import os
+import xml.etree.ElementTree as ET
+import random
+
+import kinpy as kp
+import numpy as np
+from trip_kinematics import Utility, URDFParser, Robot
+import trip_kinematics

```

Comment:
why import trip_kinematics when importing specific function from trip_kinematics at the same time